packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/CodeMirror.tsx

```diff
@@ -287,8 +287,12 @@ function CodeMirror(props: NoteBodyEditorProps, ref: any) {
 	const editorCopyText = useCallback(() => {
 		if (editorRef.current) {
 			const selections = editorRef.current.getSelections();
-			if (selections.length > 0) {
+			if (selections.length > 0 && selections[0]) {
```

Comment:
Would you mind adding a comment for the two cases please? "Copy the selection" and "Copy the current line" Since it might not be obvious what's the intent

Reply:
Sure will do that right away.

Reply:
No jsdoc please, multi-line comments only